docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
I have seen less-minimal Makefiles, e.g. https://github.com/pyca/cryptography/blob/master/docs/Makefile (I think txacme has the same one as cryptography)...not sure how those are generated (possibly just an older version of Sphinx), or if we care about the extra features.

Reply:
Let's worry about that later.